index.d.ts

```diff
@@ -8,18 +8,18 @@ export interface StoreCreator {
   ): Store<S>;
 }
 
-export type Loop<S, A extends Action> = [S, CmdType<A>];
+export type Loop<S, A extends Action = never> = [S, CmdType<A>];
 
-export interface LoopReducer<S, A extends Action = AnyAction> {
-  (state: S | undefined, action: A, ...args: any[]): S | Loop<S, A>;
+export interface LoopReducer<S, A extends Action = AnyAction, B extends Action = A> {
```

Comment:
@laszlopandy I added this so that you can optionally delineate between the set of actions that a reducer consumes vs the set of actions that will be dispatched via loops. I think that this is needed since the set of actions being dispatched may not match, or even be a subset of, the actions being consumed. The Loop actions default to the consumed actions though, so this remains backwards compatible. Thoughts?

Reply:
i think we should consider #247 when changing LoopReducer